nncf/quantization/algorithms/channel_alignment/openvino_backend.py

```diff
@@ -93,16 +93,7 @@ def get_statistic_collector(
 
     @staticmethod
     def is_node_with_bias(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
-        next_nodes = nncf_graph.get_next_nodes(node)
-        if not next_nodes:
-            return False
-
-        add_node = next_nodes[0]
-        if add_node.metatype != OVAddMetatype:
-            return False
-
-        bias_constant = get_node_with_bias_value(add_node, nncf_graph)
-        return bias_constant is not None
+        return is_node_with_bias(node, nncf_graph)
```

Comment:
It is bad practice to import a function into a module where a function with the same name exists. You can update the import as follows:
"""
from nncf.openvino.graph import node_utils
node_utils.is_node_with_bias
"""

Reply:
done